src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -76,7 +117,7 @@ If you upload artifacts **after** an error is captured by Sentry, Sentry will no
 
 ### Verify your source maps are built correctly
 
-We maintain an online validation tool that can be used to test your **hosted** source (and source maps) against: [sourcemaps.io](http://sourcemaps.io).
+We maintain an online validation tool that can be used to test your **hosted** source (and source maps): [sourcemaps.io](http://sourcemaps.io).
```

Comment:
Ohhh - now I get what the "against" was for. It was just in the wrong place, I think. Does this make more sense?

"We maintain an online validation tool that can be used to test your source maps against your **hosted** source"